packages/protocol/contracts-0.8/common/ScoreManager.sol

```diff
@@ -103,6 +141,10 @@ contract ScoreManager is
     return (1, 1, 0, 0);
   }
 
+  /**
+   * @notice Returns the actual score based on the input value.
+   * @param score The value from `validatorScores` or `groupScores` mappings .
```

Comment:
Maybe add an `@dev` explaining why this is necessary, why we treat a value of 0 as a 1.